sdk/src/oauth/state-machines/debug-oauth-2025-03-26.ts

```
@@ -479,8 +494,9 @@ export const createDebugOAuthStateMachine = (
 
           case "discovery_start":
             // Step 3: Start authorization server metadata discovery
-            const authBaseUrl = buildAuthorizationBaseUrl(state.serverUrl!);
-            const authServerUrls = buildAuthServerMetadataUrls(authBaseUrl);
+            const discoveryBaseUrl = state.serverUrl!;
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)